code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -5947,3 +5947,55 @@ obj/item/weapon/reagent_containers/food/snacks/butterfingers_l
 	..()
 	reagents.add_reagent(NUTRIMENT, 3)
 	bitesize = 2
+
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/suppermatter
+	name = "suppermatter"
+	desc = "Extremely dense and powerful food."
+	slice_path = /obj/item/weapon/reagent_containers/food/snacks/suppermattershard
+	storage_slots = 1
+	slices_num = 4
+	icon_state = "suppermatter"
+	w_class = W_CLASS_MEDIUM
```

Comment:
This needs a special eatverb "slams"

I'd like it to slam into their face but eatverb can't support complex phrasing

Reply:
`"Grey Mcshirt slams the suppermatter."`
Sounds kinda weird to me.